PWGCF/FemtoUniverse/Tasks/femtoUniverseEfficiencyBase.cxx

```diff
@@ -257,6 +291,38 @@ struct femtoUniverseEfficiencyBase {
     }
   }
 
+  bool IsParticleTPC(int pdg, float nsigmaTPC)
+  {
+    switch (pdg) {
+      case 2212:  // Proton
+      case -2212: // anty Proton
+        if (TMath::Abs(nsigmaTPC) < ConfBothTracks.ConfNsigmaTPCProton) {
```

Comment:
We have isParticleNsigma, maybe we can add it there somehow?

Reply:
I deleted IsParticleTPC and use other functions, already in the task. I also added argument of PDG code to IsParticleNSigma so it doesn't always use ConfPDGCodePartOne (for example for second track).